BEIMA.Client/src/pages/Devices/AddDevicePage.js

```diff
@@ -243,7 +277,7 @@ const AddDevicePage = () => {
           <Form >
             <Row className={styles.buttonGroup}>
               <Col>
-                <FilledDropDown dropDownText={selectedDeviceType.name} items={deviceTypes} selectFunction={getFieldsForTypeId} buttonStyle={dropDownStyle} dropDownId={"typeDropDown"} />
+                <FilledDropDown dropDownText={selectedDeviceType.name} items={deviceTypes} selectFunction={getFieldsForTypeId} buttonStyle={deviceTypeDropDownStyle} dropDownId={"typeDropDown"} />
```

Comment:
Kind of noticed this now that we have buildings as well, but maybe we should add a label for the device type and building dropdown. Since the dropdown text gets changed when a selection is made, it might be nice to have a label. But also, would a user forget what that field is for after they select it? I could go both ways about a label

Reply:
I think it's pretty standard to have a label on it. I got used to seeing the "Select &lt;what the label would be&gt; text there before integrating the API calls I didn't really think about it.
Having one only has positive effects.

Reply:
True, we should get some added then

Reply:
An alternative to the label could be an option at the very top of the selection list that says Select Building. That way they see it every time they click the select box. We can then just make sure that if they try to add with that selected they get an error/warning

Reply:
including the "select..." text in the list would require some tweaking of the way the list is generated to keep it, and the standard on the web is a label for it, and it's a much easier/quicker solution.

- [ ] add label to dropdowns for their purpose